lambdas/service/app.py

```diff
@@ -94,6 +95,10 @@
         {
             'name': 'Auxiliary',
             'description': 'Describes various aspects of the Azul service'
+        },
+        {
+            'name': 'Manifest',
+            'description': 'For requesting a manifest of files that match a given search criteria'
```

Comment:
```suggestion
            'description': 'Complete listing of files matching a given filter in TSV and other formats
```

I usually try to avoid circular definitions.